File: tests/fixtures/sample_csharp/AuthServiceTests.cs
using Sample.Services;

namespace Sample.Tests
{
    // A *Tests.cs file -> is_test_unit, so this powers suggest_tests for the symbols it exercises.
    public class AuthServiceTests
    {
        public void TestAuthenticate()
        {
            // Explicit receiver type so `service.Authenticate()` resolves (declared, not inferred).
            AuthService service = new AuthService(null);
            service.Authenticate();
        }
    }
}

File: tests/fixtures/sample_swift/AuthServiceTests.swift
// A *Tests.swift file -> is_test_unit, so it powers suggest_tests for the symbols it exercises.
class AuthServiceTests {
    func testAuthenticate() {
        let service: AuthService = AuthService()
        let user: User = User(name: "admin")
        _ = service.authenticate(u: user)
    }
}
